### PR TITLE
refactor: use material3

### PR DESCRIPTION
**Pull request by drymarau**

### CODE REVIEW

File: sample/src/main/kotlin/com/pexip/sdk/sample/conference/ConferenceCallScreen.kt [hunk 1]
```diff
@@ -39,130 +51,216 @@ import com.pexip.sdk.media.webrtc.compose.VideoTrackRenderer
...
 fun ConferenceCallScreen(
     rendering: ConferenceCallRendering,
     environment: ViewEnvironment,
     modifier: Modifier = Modifier,
 ) {
     BackHandler(onBack = rendering.onBackClick)
-    Box(
-        contentAlignment = Alignment.Center,
-        modifier = modifier.fillMaxSize(),
+    Scaffold(
```
Comment (tpettersen) [inline sample/src/main/kotlin/com/pexip/sdk/sample/conference/ConferenceCallScreen.kt:62]: image

I think I would have changed the color of the small buttons to something a bit more pexipy. 
Also, the more-dialog should have a padding from the edge IMO

File: sample/src/main/kotlin/com/pexip/sdk/sample/conference/ConferenceCallScreen.kt [hunk 2]
```diff
@@ -39,130 +51,216 @@ import com.pexip.sdk.media.webrtc.compose.VideoTrackRenderer
...
+        contentWindowInsets = WindowInsets.safeDrawing,
```
Comment (tpettersen) [inline sample/src/main/kotlin/com/pexip/sdk/sample/conference/ConferenceCallScreen.kt:63]: There is something off with the insets in landscpare. The video gets cut in the top and bottom on my phone, + it not stretching out on the left.
image
image

File: sample/src/main/kotlin/com/pexip/sdk/sample/conference/ConferenceCallScreen.kt [hunk 3]
```diff
@@ -39,130 +51,216 @@ import com.pexip.sdk.media.webrtc.compose.VideoTrackRenderer
...
+
+@Composable
+private fun DtmfItem(
+    rendering: ConferenceCallRendering,
+    onDismissRequest: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    DropdownMenuItem(
+        text = {
+            Text(text = "DTMF")
```
Comment (tpettersen) [inline sample/src/main/kotlin/com/pexip/sdk/sample/conference/ConferenceCallScreen.kt:230]:
```suggestion
            Text(text = "Keypad")
```

File: sample/src/main/kotlin/com/pexip/sdk/sample/conference/ConferenceCallScreen.kt [hunk 4]
```diff
@@ -39,130 +51,216 @@ import com.pexip.sdk.media.webrtc.compose.VideoTrackRenderer
...
-        BoxWithConstraints(
-            modifier = Modifier
-                .fillMaxSize()
-                .padding(16.dp)
-        ) {
-            val aspectRatio = remember(maxWidth, maxHeight) {
-                when {
-                    maxWidth > maxHeight -> ASPECT_RATIO_LANDSCAPE
-                    else -> ASPECT_RATIO_PORTRAIT
+                if (rendering.presentationRemoteVideoTrack != null) {
```
Comment (tpettersen) [inline sample/src/main/kotlin/com/pexip/sdk/sample/conference/ConferenceCallScreen.kt:82]: Remote presentation in landscape is kinda broken:
image

**Review by tpettersen — CHANGES_REQUESTED** [review]
Looks much nicer than before. I found some small issues and added a couple of suggestions. Take a look :)

File: gradle/libs.versions.toml [hunk 5]
```diff
@@ -1,11 +1,13 @@
...
+android = "7.3.0"
 androidx-activity = "1.5.1"
 androidx-appcompat = "1.4.2"
-androidx-compose = "1.2.0"
+androidx-compose = "1.3.0-beta02"
+androidx-compose-material3 = "1.0.0-beta02"
 androidx-datastore = "1.0.0"
 androidx-startup = "1.1.1"
-cachefix = "2.5.6"
+accompanist = "0.26.3-beta"
```
[the remark this answers is not in the document]
Comment (drymarau) [reply gradle/libs.versions.toml:9]: Don't see any problem with that

File: sample/src/main/kotlin/com/pexip/sdk/sample/conference/ConferenceCallScreen.kt (hunk 1 above)
Comment (drymarau) [reply sample/src/main/kotlin/com/pexip/sdk/sample/conference/ConferenceCallScreen.kt:62]: If your device is API31+, it'll use dynamic colors provided by your wallpaper. No plans to provide a separate branded color scheme at the moment.

Not experiencing the same issue on emulator or S10e with padding, might be fixed by updating some of the dependencies.

File: sample/src/main/kotlin/com/pexip/sdk/sample/composer/Composer.kt [hunk 6]
```diff
@@ -22,15 +27,21 @@ import androidx.compose.ui.unit.dp
 
 @Composable
 fun Composer(rendering: ComposerRendering, modifier: Modifier = Modifier) {
-    Row(modifier = modifier, verticalAlignment = Alignment.Bottom) {
+    val paddingValues = WindowInsets.safeDrawing
```
[the remark this answers is not in the document]
Comment (drymarau) [reply sample/src/main/kotlin/com/pexip/sdk/sample/composer/Composer.kt:30]: Don't think it's necessary if you look at usage examples.

**Review by tpettersen — APPROVED** [review]
https://user-images.githubusercontent.com/247298/191487428-263912db-0aaf-4075-84a5-7817c342247b.mp4

Tiniest nitpick left with the missing bottom padding, which jumps up when any button is clicked. Other than that it looks really good 👌

**Comment by drymarau** [comment]
> Tiniest nitpick left with the missing bottom padding, which jumps up when any button is clicked

Seems like `WindowInsets` are different on OnePlus devices, I'm not sure how to address it at the moment.

**Comment by tpettersen** [comment]
> > Tiniest nitpick left with the missing bottom padding, which jumps up when any button is clicked
> 
> Seems like `WindowInsets` are different on OnePlus devices, I'm not sure how to address it at the moment.

Then just skip it. Not important

**Comment by drymarau** [comment]
> Then just skip it. Not important

Do you have enable "Enable fullscreen"-like toggles enabled?

**Comment by tpettersen** [comment]
> > Then just skip it. Not important
> 
> Do you have "Enable fullscreen"-like toggles enabled?

Everything is default setup on my phone. There is a always-fullscreen-toggle I can turn on, but that is not available for the SDK app it seems.

**Pull request merged**